app/web/features/search/MapWrapper.tsx

```diff
@@ -267,6 +311,13 @@ export default function MapWrapper({
     }
   };
 
+  /**
+   * Clicks on 'clear filters' button
+   */
+  const handleClearFiltersOnClick = () => {
+    handleClearFilters();
```

Comment:
Just as a standard practice, I'd change the prop name from `handleClearFilters to `onClearFiltersClick`.

Generally the handle functions update the state in that same component. If passing down to a child component, name it the action it does instead.

Reply:
Updated! That makes sense thanks for the info.